.deepsource.toml

```diff
@@ -0,0 +1,11 @@
+version = 1
```

Comment:
Unnecessary file pushed?

Reply:
This file integrates DeepSource. It detects 560+ types of issues in the changes — including bug risks, anti-patterns, security vulnerabilities, etc. In the above PR description, you'll find links to some resources about DeepSource, the issues that it has detected in this repo, and resources that'll help you get it up and running for this repository.

Please let me know if you need anything else!